documentation/user-interface/report-view.md

```diff
@@ -35,13 +35,13 @@ TestCafe Studio provides the following report formats out of the box:
 * Minimal
 * Spec
 
-See [Get More Report Formats](#get-more-report-formats) to learn how to make more report formats available.
+See [Get More Report Formats](#get-more-report-formats) for information on how to make add more report formats.
 
-You can select the default format in [run configuration](run-configuration-dialog.md) settings.
+You can select the default format in the [run configuration](run-configuration-dialog.md) settings.
 
 ### Work with a Grid
 
-Grid view is the most powerful report format to study and analyze.
+The Grid view is the most powerful report format to study and analyze.

```

Comment:
"most powerful report format to study and analyze" This is incomplete and does not really tell the reader anything useful about the Grid View.